face/src/helpers/or.js

```diff
@@ -1,3 +1,4 @@
-export default function(a, b) {
-    return a || b;
+export default function(...args) {
+    args.pop();
+    return args.reduce((a, b) => !!(a || b), false);
```

Comment:
Handlebars by design intended not to have logic inside.
Actually, the `or` helper is an anti-pattern. While it is quite okay to break the rule for two values, it would be dangerous to make it more complex